### PR TITLE
Feature: Added transaction icons

### LINKED ISSUE #1122: Can we have different icons for Filter transactions?
**Describe the bug**
Currently we have similar icons for Coinbase, Poison-microblock and Token transfer in Filter Transaction. I think we can have different icons for each one of them. 

**To Reproduce**
1. Go to 'https://explorer.hiro.so/transactions?chain=mainnet'
2. Click on 'filters'

**Screenshots**
Screenshot 2023-05-09 at 10 34 34 PM

### PR DESCRIPTION
**Pull request by shamim-io**
closes #1122 

Feature details
Added transaction icons

Feature image
Screenshot 2023-06-19 at 11 51 45 PM

### CODE REVIEW

File: src/ui/icons/CoinbaseIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use client';
```
Comment (He1DAr) [inline src/ui/icons/CoinbaseIcon.tsx:1]: Could you please use coinbase icon from react-icons?

File: src/ui/icons/PoisonMicroblockIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+'use client';
```
Comment (He1DAr) [inline src/ui/icons/PoisonMicroblockIcon.tsx:1]: We could also use cube icon from react-icons for this one.
image

**Review by He1DAr — CHANGES_REQUESTED** [review]
Thank you so much for fixing this @shamim-io! Left a tiny nit

**Comment by shamim-io** [comment]
@He1DAr I can change the icons and use react-icons. But as per this comment: [ https://github.com/hirosystems/explorer/issues/1122#issuecomment-1592064076 ], provided figma file, and specification to use the dimension as 16 * 16. I think we might have to use the svg. 

Can you please confirm that once. Thanks!

**Comment by He1DAr** [comment]
@shamim-io We're good to use react-icons for now. Thank you @shamim-io!

**Pull request merged**